framework/src/modules/dynamic_reward/endpoint.ts

```diff
@@ -70,9 +70,9 @@ export class DynamicRewardEndpoint extends RewardEndpoint {
 			this._config,
 			defaultReward,
 		);
-		const stakeRewardActiveValidators = getStakeRewardActiveValidators(
+		const stakeRewardActiveValidators = await getStakeRewardActiveValidators(
 			context,
-			this._posMethod,
+			this._validatorMethod,
 			defaultReward,
 			minimalRewardActiveValidators,
 		);
```

Comment:
Comment about line 88: `roundLength` should be replaced by `numberActiveValidators` (see LIP PR). 

Also, in LIP we use integer division and caclulate the values manually instead of rewardPerSec, not sure if this creates some rounding missmatch, just mentioning so that you can check.

Reply:
we keep the roundLength since that's the default behavior, and this needs to count the standby delegate